api/internal/adapter/gql/gqlmodel/convert.go

```
@@ -25,29 +26,70 @@
+	}
+}
+
+func OrderDirectionToString(dir *OrderDirection) *string {
+	if dir == nil {
+		return nil
+	}
+	s := string(*dir)
+	return &s
 }
 
 func intToInt64(i *int) *int64 {
```

golangci-lint (1.62.2)

func `intToInt64` is unused

(unused)